apps/remix-ide/src/app/components/vertical-icons.tsx

```diff
@@ -0,0 +1,117 @@
+// eslint-disable-next-line no-use-before-define
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Registry from '../state/registry'
+import packageJson from '../../../../../package.json'
+import { Plugin } from '@remixproject/engine'
+import { EventEmitter } from 'events'
+import { IconRecord, RemixUiVerticalIconsPanel } from '@remix-ui/vertical-icons-panel'
+import { Profile } from '@remixproject/plugin-utils'
+import { timeStamp } from 'console'
+
+const profile = {
+  name: 'menuicons',
+  displayName: 'Vertical Icons',
+  description: '',
+  version: packageJson.version,
+  methods: ['select', 'unlinkContent', 'linkContent'],
+  events: ['toggleContent', 'showContent']
+}
+
+// TODO merge with side-panel.js. VerticalIcons should not be a plugin
```

Comment:
please create an issue for todo and add a number here

Reply:
ah old comment can be removed